src/daemon/osd_scenarios/osd_disk_activate.sh

```diff
@@ -21,8 +21,9 @@ function osd_activate {
   fi
 
   if ! test -d /etc/ceph/osd || ! grep -q ${OSD_DEVICE}1 /etc/ceph/osd/*.json; then
-    log "INFO: Scanning ${OSD_DEVICE}"
-    ceph-volume simple scan ${OSD_DEVICE}1
+    data_part=$(dev_part "${OSD_DEVICE}" 1)
+    log "INFO: Scanning ${data_part}"
+    ceph-volume simple scan ${data_part}
```

Comment:
The data_part should be set before the condition and used with the grep command.